SocketHttpListener/ByteOrder.cs

```diff
@@ -1,17 +1,17 @@
 namespace SocketHttpListener
 {
-  /// <summary>
-  /// Contains the values that indicate whether the byte order is a Little-endian or Big-endian.
-  /// </summary>
-  public enum ByteOrder : byte
-  {
     /// <summary>
-    /// Indicates a Little-endian.
+    /// Contains the values that indicate whether the byte order is a Little-endian or Big-endian.
     /// </summary>
-    Little,
-    /// <summary>
-    /// Indicates a Big-endian.
-    /// </summary>
-    Big
-  }
+    public enum ByteOrder : byte
+    {
+        /// <summary>
+        /// Indicates a Little-endian.
+        /// </summary>
+        Little,
+        /// <summary>
+        /// Indicates a Big-endian.
+        /// </summary>
+        Big
```

Comment:
another thing for another PR, but this file has way too many comments, just the class description would be enough

Reply:
Everything should be documented, In case we want to split it out as a lib.